vignettes/toInteger.Rmd

```diff
@@ -21,15 +21,29 @@ This has a couple of implications on design characteristics:
 
 This document goes through examples to demonstrate the calculations.
 The new function as of July, 2023 is the `toInteger()` which operates on group sequential designs to convert to integer-based total sample size and event counts at analyses.
-We begin with an abbreviated example for a time-to-event endpoint design to demonstrate basic concepts.
+As of November, 2024 the rounding defaults are changing as documented below.
+We begin with a summary of the method.
+Then we provide an abbreviated example for a time-to-event endpoint design to demonstrate basic concepts.
 We follow with a more extended example for a binary endpoint to explain more details.
 
+## Summary of method
+
+- With defaults, `gsSurv() |> toInteger(ratio = 0, roundUpFinal = TRUE)` 
+  - rounds event counts at interim analyses
+  - rounds event count up at final analysis
+  - rounds up sample size at final analysis, but allows continuous (expected) values at interim analyses
+- When `ratio` is a positive integer, the final sample size is rounded to a multiple of `ratio + 1`.
+  - For 1:1 randomization (experimental:control), set `ratio = 1` to round to an even sample size.
+  - For 2:1 randomization, set `ratio = 2` to round to a multiple of 3.
+  - For 3:2 randomization, set `ratio = 4` to round to a multiple of 5.
```

Comment:
For a 3:2 randomization, if users are aiming for an integer sample size and number of events, would you recommend that they input `ratio = 4` instead of `ratio = 1.5`?

Reply:
This has been documented in the updated vignette.